scripts.d/s-salome-ng

```diff
@@ -179,6 +179,15 @@ function setup-salome-ng-env()
             fi
         }
 
+        #  Patch env_launch.sh
+        #  to be able to call the extra.env.d/*.sh from env_launch.sh
+
+        cat << EOF >> env_launch.sh
+while read F; do
+    source \$F
+done < <(find \$out_dir_Path/extra.env.d/ -type f -name "*.sh")
```

Comment:
I think we need to order them properly using the number prefix. 0_ then 1_ ...etc.

Reply:
JE viens de faire une modif pour trier les numeros 
(find \$out_dir_Path/extra.env.d/ -type f -name "*.sh")  --> (find \$out_dir_Path/extra.env.d/ -type f -name "[0-9]*.sh" | sort -V)